jablib/src/main/java/org/jabref/logic/preferences/CliPreferences.java

```diff
@@ -44,6 +46,18 @@ public interface CliPreferences {
 
     void importPreferences(Path file) throws JabRefException;
 
+    UserAndHost getUserAndHost();
+
+    /** Shortcut for getUserAndHost().getUser() */
+    default String getUsername() {
+        return getUserAndHost().user();
+    }
+
+    /** Shortcut for getUserAndHost().getHost() */
+    default String getHostname() {
+        return getUserAndHost().host();
+    }
+
```

Comment:
YAGNI. This clutters CliPreferences. Too much noise. getUserAndHost().host() is simple enough.